swap_meet/item.py

```diff
@@ -0,0 +1,41 @@
+class Item():
+    def __init__(self, category = None, condition = 0):
+
+        self.condition = condition
+        if category == None:
+            self.category = ""
+        elif type(category) != str:
+            self.category = None
+        else:
+            self.category = category
+
+        self.condition = condition
```

Comment:
Since strings are immutable in Python you don't have to use the `None` trick like you do with lists:

```suggestion
    def __init__(self, category = "", condition = 0):
        self.category = category
        self.condition = condition
```